### DISCUSSION TITLE
Pluglet: mkdocs-macros-includex

### DISCUSSION THREAD

**Discussion by fralau** [fralau/mkdocs-macros-plugin#206] [Pluglet]
**Includex** (by [jannismain](https://pypi.org/user/jannismain/)):  Use includex to include anything from any file into your markdown documentation.

Page on [Pypi](https://pypi.org/project/mkdocs-macros-includex/):

Example:

```python
{{ includex("pyproject.toml", start_match="[tool.hatch.version]", code=True, lines=2, caption=True) }}
```

**Comment by jannismain** [comment]
Hey 👋

I'll just leave a link to the [mkdocs-macros-includex](https://github.com/jannismain/mkdocs-macros-includex) GitHub repository here 🙂.

Thanks for your macros plugin. It makes developing custom extensions for MkDocs much more straightforward ☺️.
